tests/integration/logging/logging.spec.js

```diff
@@ -0,0 +1,35 @@
+const utils = require('@utils');
+const moment = require('moment');
+
+const validLog = (line, before, after) => {
+  const [date, action] = line.split(/\s/);
+  console.log(date, before.toISOString(), after.toISOString());
+
+  expect(moment.utc(date, 'YYYY-MM-DDTHH:mm:ss.SSS', true).isValid()).to.equal(true);
+  expect(moment.utc(date).isBetween(before, after)).to.be.true;
+  expect(action).to.be.oneOf(['REQ:', 'RES:', 'DEBUG:', 'INFO:', 'ERROR:', 'WARN:']);
+};
+
+describe('logging', () => {
+  it('logs should include formatted date in API and Sentinel', async () => {
+    const before = moment.utc();
+    await utils.delayPromise(1000); // dates are not exact
```

Comment:
Instead of making the test execution 1 second longer, let's just do some moment date math to remove 1 second from the before variable?

```suggestion
    const before = moment.utc().subtract(1, 'second') // dates are not exact
```